apps-rendering/src/server/server.ts

```diff
@@ -240,6 +251,13 @@ async function serveRichLinkDetails(
 	res.status(200).json(response);
 }
 
+export const getPageId = (query: qs.ParsedQs): Option<string> => {
```

Comment:
I'm wondering whether a function name of `getBlockId` would be more descriptive here (and subsequent variable names of `blockId` too)?